docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,67 +2,74 @@
 title: Pod Security Standards
```

Comment:
```suggestion
title: Pod Security Standards (PSS) & Pod Security Admissions (PSA)
```

See my comment further down about the Rancher GUI.

Also:  It's just hard to talk about either resource without mentioning the other.

It's tempting to pull PSACT content into its own page, since how-to create and edit PSACTs feels like its own topic. That leaves the PSS page with the intro, a section on migration, and a section on Rancher namespace exemptions. Thoughts?

Reply:
> It's tempting to pull PSACT content into its own page, since how-to create and edit PSACTs feels like its own topic. That leaves the PSS page with the intro, a section on migration, and a section on Rancher namespace exemptions. Thoughts?

@martyav I agree, but I'm not sure what are the plans about expanding the docs further.

@jiaqiluo do you plan to address ^ in https://github.com/rancher/rancher-docs/issues/462?

Reply:
I think that a high-level section about PSS/PSA and then a subpage about PSACT would make sense.

Reply:
hi @macedogm, I agree with your idea.   
@martyav will handle #462 , so I will leave it to him to finalize the structure. 

I think we do not need to hold this PR because of that. We can get this PR merged and use #462  to do the refactor or enhancement.

Reply:
Just a note, before getting Marty's feedback about merging this PR or not, we still need to merge his suggestion above. I haven't done this yet, because it will close this thread (that is still going on).

Reply:
I think it's a good idea to merge this PR if there is nothing else to add, then I can create a working branch off of the 2.7.2 release branch, and open a PR for the PSACT how-to. It'll be easier to review the how-to that way.